src/sentry/api/endpoints/data_export.py

```diff
@@ -6,22 +6,22 @@
 from rest_framework.response import Response
 
 from sentry import features
-from sentry.api.bases.organization import OrganizationEndpoint, OrganizationEventPermission
+from sentry.api.bases.organization import OrganizationEndpoint, OrganizationDataExportPermission
 from sentry.api.serializers import serialize
 from sentry.constants import ExportQueryType
 from sentry.models import ExportedData
 from sentry.tasks.data_export import assemble_download
 
 
 class ExportedDataSerializer(serializers.Serializer):
-    max_value = len(ExportQueryType.as_choices()) - 1
-    query_type = serializers.IntegerField(required=True, min_value=0, max_value=max_value)
-    query_info = serializers.JSONField(required=True)
+
+    query_type = serializers.ChoiceField(choices=ExportQueryType.as_str_choices(), required=True)
```

Comment:
I don't know if this comment already got added in my review lol but I think choices should be a tuple (even though _technically_ it can be a list of values I think)

Anyway, we should do something like:
https://github.com/getsentry/sentry/blob/d46213a3cb4fea1653724ac58bcd78e32a3dda05/src/sentry/api/validators/monitor.py#L46

Reply:
Done! and I think it addresses your other comment as well. I agree its kinda ugly and a dictionary would be a lot nicer, but I kept the methodology from here:
https://github.com/getsentry/sentry/blob/d46213a3cb4fea1653724ac58bcd78e32a3dda05/src/sentry/constants.py#L394-L417